tests/features/support/env.rb

```diff
@@ -28,6 +28,10 @@
   skip_this_scenario("Skipping scenario") unless %w[ANDROID_8_0 ANDROID_8_1 ANDROID_9_0 ANDROID_10_0].include? bs_device
 end
 
+Before('@skip_android_8_1') do |scenario|
+  skip_this_scenario("Skipping scenario") unless %w[ANDORID_8_1].include? bs_device
```

Comment:
I think you what an "if", not "unless".  But I'd also be interested to know why we want to skip on 8.1 specifically (it is soft fail anyway, for example).

Reply:
Due to the timings of the ANRs triggered on the Samsung Galaxy device we're using for 8.1 they will never pass our current tests.  While we could leave it to soft fail, as this is a known issue I'm comfortable with skipping this particular combination and allowing the test to run the through the rest of the scenarios.

Reply:
Makes sense, could we just write a short comment to that effect?  Especially as it's due to the specific device, as opposed to 8.1 in general.